sentiment_analysis_model/sentiment_analysis_model.py

```diff
@@ -0,0 +1,150 @@
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+from nltk.stem.porter import PorterStemmer
+from nltk.stem import WordNetLemmatizer
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import MultinomialNB
+
+import string
+
+import pandas as pd
+
+
+df = pd.read_csv('Womens Clothing Reviews.csv')
```

Comment:
split code by modules. For example:
1. train_model.py
2. data_handler.py
3. utils.

Reply:
will be good to add settings file